resources/views/layouts/empty.blade.php

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
you dont need to include livewire styles and scripts in livewire v3.
also dont use the vite helper, better use:
```
    {{ Asset::toHtml(
            ViewServiceProvider::getRealPackageAssetPath(
                '/resources/js/alpine.js',
                'team-nifty-gmbh/flux-erp'
            )
        )
    }}
```

Reply:
livewire loaded twice using this approach